docker-to-ecr.sh

```diff
@@ -3,9 +3,7 @@
 set -e
 
 # get aws account id if it does not exist in env var
-AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID:-$(aws sts get-caller-identity --query "Account")}
-AWS_ACCOUNT_ID="${AWS_ACCOUNT_ID%\"}"
-AWS_ACCOUNT_ID="${AWS_ACCOUNT_ID#\"}"
+AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID:-$(aws sts get-caller-identity --query "Account" --output text)}
```

Comment:
Does the text output get logged anywhere?

Reply:
This script is only run from our local workstations.

Reply:
I noticed this `--output text` command in a lot of other places in the code changes for Sprint 50, including https://github.com/ustaxcourt/ef-cms/blob/6001af40baed6cb7fc072f14467c6000465e803e/set-tokens.sh 
which is run in CircleCI. If this outputs text to logs, we should change that anywhere it will be run in Circle or AWS environment, I think.

Reply:
Eric looked into this and saw that CircleCI shows no output for these assignments. Good to know!